README.md

```diff
@@ -47,11 +47,11 @@ for various platforms:
 
 | **Platform** | **Status** |
 |---|---|
-| **macOS** | Supported |
+| **macOS** | [![Build Status](https://ci.swift.org/buildStatus/icon?job=pr-swift-testing-macos)](https://ci.swift.org/job/pr-swift-testing-macos/) |
 | **iOS** | Supported |
 | **watchOS** | Supported |
 | **tvOS** | Supported |
-| **Ubuntu 22.04** | Supported |
+| **Ubuntu 22.04** | [![Build Status](https://ci.swift.org/buildStatus/icon?job=pr-swift-testing-linux-main)](https://ci.swift.org/job/pr-swift-testing-linux-main/) |
```

Comment:
I wonder if we should break out these badges into a third column in the table, so we can still show their level of support status?

Also, the title of this section is currently "Supported platforms" but I wonder if now we ought to expand that to a broader title. Some other projects display their CI status more prominently towards the top of the README, and I thought once we began including CI we might do that as well.

Reply:
I can put it in a third "Build Status" column maybe?

I don't have any preferences about the title of the section or its position.

Reply:
Sure - how about these for the column titles?
```
| **Platform** | **CI Status** | **Support Status** |
```

Reply:
I buy it.